Integration test failure in `api/tests/integration_tests/tools/`

### Self Checks

- [x] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/langgenius/dify/discussions/categories/general).
- [x] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [x] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [x] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:)
- [x] Please do not modify this template :) and fill in all the required fields.

### Dify version

1.3.1

### Cloud or Self Hosted

Self Hosted (Source)

### Steps to reproduce

Run `pytest api/tests/integration_tests/tools/` on the main branch.

### ✔️ Expected Behavior

The integration test should pass.

### ❌ Actual Behavior

Error message:

```
api\tests\integration_tests\tools\api_tool\test_api_tool.py F                                                                 [100%]

============================================================= FAILURES =============================================================
__________________________________________________________ test_api_tool ___________________________________________________________

setup_http_mock = None

    def test_api_tool(setup_http_mock):
        tool = ApiTool(
            entity=ToolEntity(
>               identity=ToolIdentity(provider="", author="", name="", label=I18nObject()),
            ),
            api_bundle=ApiToolBundle(**tool_bundle),
            runtime=ToolRuntime(tenant_id="", credentials={"auth_type": "none"}),
        )

api\tests\integration_tests\tools\api_tool\test_api_tool.py:37:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _  

self = I18nObject(), data = {}

    def __init__(self, **data):
>       super().__init__(**data)
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for I18nObject
E       en_US
E         Field required [type=missing, input_value={}, input_type=dict]
E           For further information visit https://errors.pydantic.dev/2.9/v/missing

api\core\tools\entities\common_entities.py:17: ValidationError
```

--- a/.github/workflows/api-tests.yml
+++ b/.github/workflows/api-tests.yml
@@ -88,3 +88,6 @@ jobs:
 
       - name: Run Workflow
         run: uv run --project api bash dev/pytest/pytest_workflow.sh
+
+      - name: Run Tool
+        run: uv run --project api bash dev/pytest/pytest_tools.sh

--- a/api/tests/integration_tests/tools/__mock/http.py
+++ b/api/tests/integration_tests/tools/__mock/http.py
@@ -5,6 +5,8 @@ import httpx
 import pytest
 from _pytest.monkeypatch import MonkeyPatch
 
+from core.helper import ssrf_proxy
+
 
 class MockedHttp:
     @staticmethod
@@ -29,6 +31,6 @@ class MockedHttp:
 
 @pytest.fixture
 def setup_http_mock(request, monkeypatch: MonkeyPatch):
-    monkeypatch.setattr(httpx, "request", MockedHttp.httpx_request)
+    monkeypatch.setattr(ssrf_proxy, "make_request", MockedHttp.httpx_request)
     yield
     monkeypatch.undo()

--- a/api/tests/integration_tests/tools/api_tool/test_api_tool.py
+++ b/api/tests/integration_tests/tools/api_tool/test_api_tool.py
@@ -34,10 +34,11 @@ parameters = {
 def test_api_tool(setup_http_mock):
     tool = ApiTool(
         entity=ToolEntity(
-            identity=ToolIdentity(provider="", author="", name="", label=I18nObject()),
+            identity=ToolIdentity(provider="", author="", name="", label=I18nObject(en_US="test tool")),
         ),
         api_bundle=ApiToolBundle(**tool_bundle),
         runtime=ToolRuntime(tenant_id="", credentials={"auth_type": "none"}),
+        provider_id="test_tool",
     )
     headers = tool.assembling_request(parameters)
     response = tool.do_http_request(tool.api_bundle.server_url, tool.api_bundle.method, headers, parameters)

--- a/dev/pytest/pytest_tools.sh
+++ b/dev/pytest/pytest_tools.sh
@@ -1,4 +1,4 @@
 #!/bin/bash
 set -x
 
-pytest api/tests/integration_tests/tools/test_all_provider.py
+pytest api/tests/integration_tests/tools

